### PR TITLE
README needed some dusting and cleanup

### PR DESCRIPTION
**Pull request by dhruvbatra**
- moving details out to another file
- updating readme

## Types of changes

- Docs change / refactoring / dependency upgrade
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: configs/tasks/rearrange/play.yaml [hunk 1]
```diff
@@ -170,7 +170,7 @@ SIMULATOR:
     DEFAULT_AGENT_ID: 0
     FORWARD_STEP_SIZE: 0.25
     HABITAT_SIM_V0:
-        ALLOW_SLIDING: False
+        ALLOW_SLIDING: True
```
Comment (dhruvbatra) [inline configs/tasks/rearrange/play.yaml:173]: @ASzot -- you okay with this?

File: README.md [hunk 2]
```diff
@@ -67,171 +57,154 @@ If you use the Habitat platform in your research, please cite the [Habitat](http
...
+    ```bash
+    [16:11:11:718584]:[Sim] Simulator.cpp(205)::reconfigure : CreateSceneInstance success == true for active scene name : data/scene_datasets/habitat-test-scenes/skokloster-castle.glb  with renderer.
+    2022-08-13 16:26:45,068 Initializing task Nav-v0
+    Environment creation successful
+    Agent stepping around inside environment.
+    Episode finished after 5 steps.
+    ```
 
-Habitat Lab documentation is available [here](https://aihabitat.org/docs/habitat-lab/index.html).
+1. **Non-interactive testing**: Test the Pick task: Run the example pointnav script
```
Comment (ASzot) [inline README.md:123]:
```suggestion
1. **Non-interactive testing**: Test the Pick task: Run the example pick task script
```

File: README.md [hunk 3]
```diff
@@ -67,171 +57,154 @@ If you use the Habitat platform in your research, please cite the [Habitat](http
...
 
-* `Env`: the fundamental environment concept for Habitat. All the information needed for working on embodied tasks with a simulator is abstracted inside an Env. This class acts as a base for other derived environment classes. Env consists of three major components: a Simulator, a Dataset (containing Episodes), and a Task, and it serves to connects all these three components together.
+1. **Interactive testing**: Using you keyboard and mouse to control a Fetch robot in a ReplicaCAD environment:
+    ```bash
+    # Pygame for interactive visualization, pybullet for inverse kinematics
+    pip install pygame==2.0.1 pybullet==3.0.4
 
-* `Dataset`: contains a list of task-specific episodes from a particular data split and additional dataset-wide information. Handles loading and saving of a dataset to disk, getting a list of scenes, and getting a list of episodes for a particular scene.
+    # Interactive play script
+    python examples/interactive_play.py --never-end --add-ik
```
Comment (ASzot) [inline README.md:162]:
```suggestion
    python examples/interactive_play.py --never-end
```
I think by default no ik so people don't have to install PyBullet.

File: configs/tasks/rearrange/play.yaml (hunk 1 above)
Comment (ASzot) [reply configs/tasks/rearrange/play.yaml:173]: Yeah this is a good idea.

File: README.md (hunk 3 above)
Comment (dhruvbatra) [reply README.md:162]: I tried playing around with both. It's very hard to control the arm without IK. Why the hesitation to pip install pybullet?

Comment (ASzot) [reply README.md:162]: Just an additional command they need to run to get things working, but I agree it's not a big deal to install pybullet.

**Review by ASzot — APPROVED** [review]

**Pull request merged**